### DISCUSSION TITLE
Issue trying to reuse an assignment

### DISCUSSION THREAD

**Discussion by FabianMorenoSyntax** [community/community#137695] [GitHub Education]
### Hi everyone,

I hope this is the right place to ask 🙂

We have an issue when trying to reuse an assignment from one GH Classroom in another GH Classroom (different Org). 
The Assignment itself gets copied normally, but the starter code Repository won't get copied correctly:

Instead of copying it and setting it as starter code template in the assignment settings, it gets copied many times (→ img) but without actually connecting it to the assignment (The message is displayed since yesterday → img). 

Our students can still accept the assignment, but they will get an empty repository.

Any ideas on how to fix that? Thank you!

<img width="100%" alt="Bildschirmfoto 2024-09-04 um 10 14 02" src="https://github.com/user-attachments/assets/d151ef21-579d-43a9-8082-c5e0394b4c75">
<img width="100%" alt="Bildschirmfoto 2024-09-04 um 10 14 16" src="https://github.com/user-attachments/assets/d5b39c6d-8da3-436f-a570-0e98c32a1d28">

**Comment by andrefqms** [comment]
This you mean ? 

https://github.com/github/docs/blob/main/content/education/manage-coursework-with-github-classroom/teach-with-github-classroom/reuse-an-assignment.md

**Reply by FabianMorenoSyntax** [reply]
Yes! It doesn't work for us
